prelude/cxx/tools/make_comp_db.py

```diff
@@ -54,8 +54,16 @@ def merge(args):
 
     entries = []
     for entry in args.entries:
-        with open(entry) as f:
-            entries.append(json.load(f))
+        if entry.startswith('@'):
```

Comment:
Is this actually required, given that we have support for `@` files in `gen`?

Reply:
The two modes that this script can run in don't share any code at all, so some argfile handling mechanism is necessary here as well.

We could lift it into a small utility function.  Basically a function that goes from an argv that potentially contains an argsfile reference to one that has them all expanded.

Would that be better?

Reply:
Given it's two entirely separate functions that just happen to live in the same file, I'd just leave it for now. I'll land this as is, and we can potentially common up later.